navigator.lua

```diff
@@ -191,6 +191,24 @@ function navup()
   handler()
 end
 
+function movepageup()
+  if cursor == 0 then return end
+  local prev_cursor = cursor
+  cursor = cursor - settings.visible_item_count
+  if cursor < 0 then cursor = 0 end
+  if selection then mp.commandv("playlist-move", prev_cursor, cursor) end
```

Comment:
This line from playlistmanager is probably something that filenavigator shouldn't do 🤔 (same for pagedown)

Reply:
line removed